cmd/talosctl/cmd/talos/list.go

```diff
@@ -45,7 +45,7 @@ var lsCmd = &cobra.Command{
 				rootDir = args[0]
 			}
 
-			stream, err := c.LS(ctx, machineapi.ListRequest{
+			stream, err := c.LS(ctx, &machineapi.ListRequest{
```

Comment:
Linter complained about a context being passed by value that was part of the `ListRequest`. Passing by reference fixes this.